[2023-10-29T15:33:14.414+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task manual__2023-10-29T15:33:09.189348+00:00 [queued]>
[2023-10-29T15:33:14.466+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task manual__2023-10-29T15:33:09.189348+00:00 [queued]>
[2023-10-29T15:33:14.470+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-10-29T15:33:14.742+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2023-10-29 15:33:09.189348+00:00
[2023-10-29T15:33:14.800+0000] {standard_task_runner.py:57} INFO - Started process 958 to run task
[2023-10-29T15:33:14.924+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'manual__2023-10-29T15:33:09.189348+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp6aerqj3n']
[2023-10-29T15:33:15.019+0000] {standard_task_runner.py:85} INFO - Job 132: Subtask second_task
[2023-10-29T15:33:15.741+0000] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v5.second_task manual__2023-10-29T15:33:09.189348+00:00 [running]> on host 9bb87dc64328
[2023-10-29T15:33:17.023+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-10-29T15:33:09.189348+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-29T15:33:09.189348+00:00'
[2023-10-29T15:33:17.027+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-29T15:33:17.030+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2023-10-29T15:33:17.104+0000] {subprocess.py:86} INFO - Output:
[2023-10-29T15:33:17.112+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2023-10-29T15:33:17.115+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-29T15:33:17.282+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20231029T153309, start_date=20231029T153314, end_date=20231029T153317
[2023-10-29T15:33:17.532+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-29T15:33:18.514+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
